src/eventloop.jl

```diff
@@ -18,9 +18,14 @@ function eventloop(socket)
                 end
             finally
                 @async begin
-                    sleep(idle_delay[])
+                    idle_ready_event += 1
```

Comment:
Shouldn't the `idle_ready_event` be initialised to 1, and this be `-= 1` and the other one below increment it? This needs to notify to anyone waiting on `idle_ready_event` no?